package.json

```
@@ -6,11 +6,14 @@
   "scripts": {
     "lint": "pnpm -r lint",
     "lint:ci": "pnpm -r lint:ci",
-    "changeset": "changeset"
+    "changeset": "changeset",
+    "release": "changeset publish",
+    "packages:build": "pnpm -r --filter=\"@ensnode/*\" --filter=\"!@ensnode/shared-configs\" build"
```

Perhaps if we define `prepublish` only on the packages that we want to include in the release, we wouldn't need to filter any packages out
```suggestion
    "packages:prepublish": "pnpm -r --filter='@ensnode/*' prepublish"
```

---

I've actually tested it out:

<details><summary>View console output</summary>

```bash
pnpm -r --filter='@ensnode/*' prepublish
Scope: 7 of 12 workspace projects
packages/ens-deployments prepublish$ tsup
[1 lines collapsed]
│ CLI Using tsconfig: tsconfig.json
│ CLI tsup v8.3.6
│ CLI Using tsup config: /Users/tko/dev/github/namehash/ensnode/packages/ens-deployments/tsup.config.ts
│ CLI Target: es2022
│ CLI Cleaning output folder
│ ESM Build start
│3ESM dist/index.js     234.19 KB
│ CLI Using tsup config: /Users/tko/dev/github/namehash/ensnode/packages/ens-deployments/tsup.config.ts
│ CLI Target: es2022
│ CLI Cleaning output folder
└─ Done in 4.2srt
│ ESM dist/index.js     234.19 KB
│ ESM dist/index.js.map 441.77 KB
│ ESM ⚡️ Build success in 24ms
│ DTS Build start
│ DTS ⚡️ Build success in 3120ms
│ DTS dist/index.d.ts 1.13 MB
│ ESM dist/subname-helpers.js.map 7.45 KB
│ ESM ⚡️ Build success in 9ms
│ DTS Build start
│ DTS ⚡️ Build success in 1871ms
│ DTS dist/subname-helpers.d.ts 2.35 KB
│ DTS dist/types.d.ts           631.00 B
└─ Done in 3s
packages/ponder-schema prepublish$ tsup
│ CLI Building entry: src/ponder.schema.ts
│ CLI Using tsconfig: tsconfig.json
│ CLI tsup v8.3.6
│ CLI Using tsup config: /Users/tko/dev/github/namehash/ensnode/packages/ponder-schema/tsup.config.ts
│ CLI Target: es2022
│ CLI Cleaning output folder
│ ESM Build start
│ ESM dist/ponder.schema.js     16.75 KB
│ ESM dist/ponder.schema.js.map 30.05 KB
│ ESM ⚡️ Build success in 8ms
└─ Done in 1s
packages/ponder-subgraph-api prepublish$ tsup
[9 lines collapsed]
│ ESM dist/helpers.js        299.00 B
│ ESM dist/graphql.js.map    57.48 KB
│ ESM dist/middleware.js.map 60.73 KB
│ ESM dist/helpers.js.map    680.00 B
│ ESM ⚡️ Build success in 13ms
│ DTS Build start
│ DTS ⚡️ Build success in 2489ms
│ DTS dist/middleware.d.ts 627.00 B
│ DTS dist/graphql.d.ts    2.20 KB
│ DTS dist/helpers.d.ts    148.00 B
└─ Done in 3.8s
packages/ensrainbow-sdk prepublish$ tsup
[15 lines collapsed]
│ ESM dist/client.js.map      11.08 KB
│ ESM dist/index.js.map       12.74 KB
│ ESM ⚡️ Build success in 8ms
│ DTS Build start
│ DTS ⚡️ Build success in 1468ms
│ DTS dist/index.d.ts       304.00 B
│ DTS dist/client.d.ts      7.38 KB
│ DTS dist/consts.d.ts      338.00 B
│ DTS dist/labelUtils.d.ts  397.00 B
│ DTS dist/label-utils.d.ts 98.00 B
└─ Done in 2s
```
</details>

---

@tk-o smart!